eogrow/pipelines/testing.py

```diff
@@ -47,11 +47,11 @@ def run_procedure(self) -> Tuple[List, List]:
 
         self.area_manager.get_area_geometry()
         grid = self.area_manager.get_grid()
-        num_patches = sum([len(df.index) for df in grid])
+        num_patches = sum([len(df.index) for df in grid.values()])
```

Comment:
`len` works also on the df itself, so it could be just 

```suggestion
        num_patches = sum(map(len, grid.values()))
```